src/Storages/System/StorageSystemDictionaries.cpp

```diff
@@ -70,96 +135,84 @@ void StorageSystemDictionaries::fillData(MutableColumns & res_columns, ContextPt
     if (!check_access_for_dictionaries)
         return;
 
-    for (const auto & load_result : external_dictionaries.getLoadResults())
+    for (size_t index = 0; const auto & load_result : external_dictionaries.getLoadResults())
```

Comment:
`external_dictionaries.getLoadResults()` returns vector. We can just write index loop over it. Without introducing additional `index` variable.

Reply:
Changed the code. No longer relevant